src/table.rs

```diff
@@ -24,8 +24,8 @@ mod tests;
 enum MmapFile {
     File {
         name: PathBuf,
-        // TODO: remove this mutex and allow multi-thread read
-        file: Mutex<fs::File>,
+        file: fs::File,
```

Comment:
Why still keeping it?

Reply:
`Mmap` only needs reference to a file (as the OS just needs the file handler to create a mmap file). Therefore, we have to own the File struct inside `MmapFile`.